src/level.h

```diff
@@ -2690,34 +2690,34 @@ struct Level : IGame {
             }
 
             Core::pass = Core::passCompose;
-            /*
+            
             if (view == 0 && Input::hmd.ready) {
```

Comment:
try to move this code into platform/win/main.cpp with Core::eyeTex's

Reply:
Is the `view == 0` important. If so that might make it a little tricky to move.